decidim-core/app/views/decidim/messaging/conversations/_start.html.erb

```diff
@@ -1,17 +1,9 @@
-<% add_decidim_page_title(t(".title")) %>
-
-<div class="add-message">
-  <h2 class="section-heading">
-    <%= t(".title") %>
-  </h2>
-
-  <%= decidim_form_for form, url: decidim.conversations_path, remote: true do |f| %>
-    <% @form.recipient.each do |recipient| %>
-      <%= f.hidden_field :recipient_id, id: nil, name: "conversation[recipient_id][]", value: recipient.id %>
-    <% end %>
-    <%= f.label :body, nil, class: "show-for-sr" %>
-    <%= f.text_area :body, label: false, rows: 4, required: true, data: { input_emoji: true } %>
-
-    <%= f.submit t(".send") %>
-  <% end %>
-</div>
+<%# REDESIGN_PENDING: new conversation throws an error %>
```

Comment:
I think this issue should be fixed before we merge this PR, even if the modal will be handled later.

The error it showed in the response when the conversation started was "Recipient can't be blank". Which I guess is because the conversation starting form does not pass the `conversation[recipient_id][]` values correctly to the `create` action.

Reply:
Fixed